front-end/src/components/newsfeed-view/news-list/news-card/InstagramNewsCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Timestamp from './elements/Timestamp';
+import './newscard.css';
+
+const InstagramNewsCard = ({data}) => (
```

Comment:
can we break this component into smaller pieces?
We can have a header, then image box, and other things, instead of jamming everything together